src/content/developers/tutorials/erc20-annotated-code/index.md

```diff
@@ -0,0 +1,907 @@
+---
+title: "ERC20 Contract Walk-Through"
```

Comment:
Hyphenated "ERC-20" (throughout) =)
(per https://eips.ethereum.org/EIPS/eip-20)

Reply:
I changed it where it isn't a code quote.